css/main.css

```diff
@@ -0,0 +1,589 @@
+@import url("https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto:wght@700&family=Inter:wght@400;700&display=swap");
+
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+html {
+  scroll-behavior: smooth;
+}
+
+a {
+  text-decoration: none;
+}
+
+hr {
+  width: 80vw;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 15px;
+  background-color: #3c3a39;
+  color: white;
+}
+
+.mobile-nav {
+  display: none;
+  position: absolute;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+  background-color: #3c3a39;
+  z-index: 100;
+}
+
```

Comment:
- [optional] Please make your menu like the demo as much as possible
- like you need to make the menu items to the left, change their colors to white also change the background color to make it like the demo.

- Your menu is like this 
![Screenshot from 2022-04-14 22-42-11](https://user-images.githubusercontent.com/33752735/163472183-493b1811-1a47-498b-9bbc-20013b381ecf.png)

- But is should be like this 
![Screenshot from 2022-04-13 21-52-09](https://user-images.githubusercontent.com/33752735/163472255-230ccb31-cead-4944-a219-fd4e02560789.png)